XO.py

```diff
@@ -0,0 +1,157 @@
+from random import shuffle
+
+# size of board
+N = 3
+
+# Utility functions
+
+
+def print_board(board):
+    print()
+    print('Board:')
+    for line_ in board:

```

Comment:
Why `line_`?

Reply:
I tried to avoid shadowing. But this is not necessary. Fixed.